test/utils/expected-files.js

```diff
@@ -209,9 +209,9 @@ const expectedFiles = {
     ],
 
     userManagementServer: [
-        `${SERVER_MAIN_RES_DIR}config/liquibase/authorities.csv`,
-        `${SERVER_MAIN_RES_DIR}config/liquibase/users.csv`,
-        `${SERVER_MAIN_RES_DIR}config/liquibase/users_authorities.csv`,
+        `${SERVER_MAIN_RES_DIR}config/liquibase/authority.csv`,
+        `${SERVER_MAIN_RES_DIR}config/liquibase/user.csv`,
+        `${SERVER_MAIN_RES_DIR}config/liquibase/user_authority.csv`,
```

Comment:
Those 3 csv files are expected to be in the `config/liquibase/data/` directory no?